src/System.Private.ServiceModel/tests/Scenarios/Binding/UDS/Binding.UDS.IntegrationTests.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="CoreWCF.UnixDomainSocket" Version="1.5.0-preview1" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.0" />
```

Comment:
Use [2.1.34](https://www.nuget.org/packages/Microsoft.AspNetCore.Http/2.1.34) instead as the 2.2.x versions are out of support.

Reply:
Thanks @mconnew for the review! I tried 2.1.34, got a build error detecting package downgrade, any suggestion?
```
error NU1605:  Binding.UDS.IntegrationTests -> CoreWCF.UnixDomainSocket 1.5.0-preview1 -> Microsoft.AspNetCore.Server.Kestr
el.Core 2.2.0 -> Microsoft.AspNetCore.Http (>= 2.2.0)
```

Reply:
Put in a package dependency override for Kestrel too with the latest 2.1.x version. The GA version of CoreWCF doesn't have any of these problems so it's temporary.

Reply:
In that case, a direct dependency of `CoreWCF.UnixDomainSocket` package downgrade is detected  :(
```
c:\wcf\src\System.Private.ServiceModel\tests\Scenarios\Binding\UDS\Binding.UDS.IntegrationTests.csproj : error NU1605:
Binding.UDS.IntegrationTests -> 
  CoreWCF.UnixDomainSocket 1.5.0-preview1 ->
    Microsoft.AspNetCore.Server.Kestrel.Core (>= 2.2.0) 
```

The corresponding package reference:
```
  <ItemGroup>
    <PackageReference Include="CoreWCF.UnixDomainSocket" Version="1.5.0-preview1" />
    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.1.34" />
    <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Core" Version="2.1.25" />
    <FrameworkReference Include="Microsoft.AspNetCore.App" />
  </ItemGroup>
```